v2.5.x/README.md

```diff
@@ -0,0 +1,77 @@
+# SOF-2.5 prebuilt binaries for Intel cAVS2.5 hardware
+
+**IMPORTANT NOTICE**
+This binary release is an opt-in release for Intel cAVS2.5 hardware
+(11th, 12th and 13th Generation Intel® Core™ Processors).
+
+Notably the Linux kernel requirements of this release are more strict
+and the provided set of DSP topology files do not cover all the hardware
+that is supported by stable releases.
+
+For a complete release that includes support for all hardware, you MUST
+use official release tarballs (currently SOF2.2) found here:
+
+https://github.com/thesofproject/sof-bin/releases
+
+## Requirements
+
+### Linux Kernel
+
+These prebuilt binaries use the new SOF IPC4 host interface
+and thus a recent Linux kernel version is required. All the required
+patches are submitted to inclusion into mainline Linux kernel and are
+queued for later release in Linux 6.4. Before 6.4 is out (expected to
+be out in summer of 2023), following pre-6.4 kernel snapshots have been
+validated to work with SOF2.5 release for cAVS2.5:
+
+Linux-next 20230320 or newer
+https://git.kernel.org/pub/scm/linux/kernel/git/next/linux-next.git/tag/?h=next-20230320
+
+ASoC upstream tree, staging branch for 6.4
+https://git.kernel.org/pub/scm/linux/kernel/git/broonie/sound.git/commit/?id=37b58becc1cee4d591024f2056d7ffa99c6089e0
+
+SOF Linux kernel staging for 6.4
+https://github.com/thesofproject/linux/commit/8525b92425e781dfef8c4344e4008e850a304395
+
+### Hardware Capabilities
+
+This release includes a set of DSP topology files that cover a large
+set of systems equipped with a HDA audio codec for headset and speakers,
+and a set of digital microphones connected directly to the Platform
+Controller Hub (PCH).
+
+## Installing the SOF binaries
+
+Use the install script at sof-bin/install.sh to install
+the binaries on your system (or follow the manual instructions
+described in the install.sh file).
+
+## Enabling SOF2.5 on cAVS2.5
+
+The SOF driver will by default look for a stable firmware,
+so following additional steps are needed to take the new
+firmware into use:
+
+```
+ln -s /lib/firmware/intel/sof-v2.5/* /lib/firmware/intel/avs/
+ln -s /lib/firmware/intel/sof-tplg-v2.5/* /lib/firmware/intel/avs-tplg/
```

Comment:
@ujfalusi /lib/firmware/intel/avs/ may exist on systems and we don't want to replace it completely as it may remove distribution controlled files. For avs-tplg this would work.

Reply:
What can install it, it was not in there on any of my systems, so it needs to be created first then?

Reply:
@ujfalusi Added some extra steps to work with and without other avs binaries installed on the system.